pkg/api/componentreadiness/test_details.go

```diff
@@ -30,6 +31,9 @@ func GetTestDetails(ctx context.Context, client *bigquery.Client, prowURL, gcsBu
 		gcsBucket:  gcsBucket,
 		ReqOptions: reqOptions,
 	}
+	if os.Getenv("DEV_MODE") == "1" {
```

Comment:
I wish I could suggest something more elegant here.   Thought about tucking this into GetLastReportModifiedTime that is used in component_report or a way to override cacheDuration.  

I get you just want to disable the caching for the report rendering and not the underlying queries and this seems effective.  Maybe something for another day.